src/periphery/SablierMerkleFactory.sol

```diff
@@ -1,23 +1,49 @@
-// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-License-Identifier: BUSL-1.1
 pragma solidity >=0.8.22;
 
 import { uUNIT } from "@prb/math/src/UD2x18.sol";
 
+import { Adminable } from "../core/abstracts/Adminable.sol";
 import { ISablierLockupLinear } from "../core/interfaces/ISablierLockupLinear.sol";
 import { ISablierLockupTranched } from "../core/interfaces/ISablierLockupTranched.sol";
 
+import { ISablierMerkleBase } from "./interfaces/ISablierMerkleBase.sol";
 import { ISablierMerkleFactory } from "./interfaces/ISablierMerkleFactory.sol";
 import { ISablierMerkleInstant } from "./interfaces/ISablierMerkleInstant.sol";
 import { ISablierMerkleLL } from "./interfaces/ISablierMerkleLL.sol";
 import { ISablierMerkleLT } from "./interfaces/ISablierMerkleLT.sol";
 import { SablierMerkleInstant } from "./SablierMerkleInstant.sol";
 import { SablierMerkleLL } from "./SablierMerkleLL.sol";
 import { SablierMerkleLT } from "./SablierMerkleLT.sol";
-import { MerkleBase, MerkleLL, MerkleLT } from "./types/DataTypes.sol";
+import { MerkleBase, MerkleFactory, MerkleLL, MerkleLT } from "./types/DataTypes.sol";
 
 /// @title SablierMerkleFactory
 /// @notice See the documentation in {ISablierMerkleFactory}.
-contract SablierMerkleFactory is ISablierMerkleFactory {
+contract SablierMerkleFactory is
+    ISablierMerkleFactory, // 2 inherited components
+    Adminable // 1 inherited component
+{
+    /*//////////////////////////////////////////////////////////////////////////
+                                  STATE VARIABLES
+    //////////////////////////////////////////////////////////////////////////*/
+
+    /// @inheritdoc ISablierMerkleFactory
+    uint256 public defaultSablierFee;
+
+    /// @dev A mapping of custom Sablier fees by user.
+    mapping(address campaignCreator => MerkleFactory.SablierFee customFee) private _sablierFeeByUser;
```

Comment:
Agree. Will change it to  `_sablierFeeByUsers` and `MerkleFactory.SablierFeeByUser`.

Its declared `private` to signal that its not used in any of the child contracts i.e. its private to factory contract only. Why would you mark it to `internal`?

Reply:
> Its declared `private` to signal that its not used in any of the child contracts i.e. its private to factory contract only. Why would you mark it to `internal`?

the only argument is that we used to be consistent with `internal` over everything, but we can change this to no longer be. i agree with this.

> _deployMerkleLT as private instead of internal?

i agree with this as well

Reply:
Not everything. [Helper contract has private functions](https://github.com/sablier-labs/v2-core/blob/main/src/libraries/Helpers.sol#L217). But may be thats an exception.

Regardless, since you also agree with this, sounds good to me. We can use `internal` for functions/variables that can be overriden by the child contract and `private` for functions/variables are not supposed to be overriden by the child contracts.